API.md

```diff
@@ -378,6 +379,26 @@ const App = () => {
     .
 }
 ```
+---
+
+### checkActiveSession
```

Comment:
Can we use a different name for this method as `checkActiveSession` means that the method will only check if there is an active session and do nothing else? Since this method also will sign the user in, I guess we better have a name that will imply that as well.

Reply:
changed with config name to `autoSignIn`